Sources/SKCore/Platform+CurrentConcurrencySafe.swift

```diff
@@ -0,0 +1,71 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2018 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import enum PackageLoading.Platform
+import struct TSCBasic.AbsolutePath
+import protocol TSCBasic.FileSystem
+import class TSCBasic.Process
+import var TSCBasic.localFileSystem
+
+// Copy pasted from PackageLoading/Platform.swift to make it concurrency-safe
+
+extension Platform {
+  // This is not just a computed property because the ToolchainRegistryTests
+  // change the value.
+  public nonisolated(unsafe) static let currentConcurrencySafe: Platform? = {
```

Comment:
Can't we fix this in SwiftPM? From the sounds of the comment, the SwiftPM one only needed to be a var for the test that's being removed here

Reply:
Oh, indeed. I just assumed that tests in SwiftPM would also modify it. But we are indeed the only ones that require it to be a variable.